view/frontend/web/template/payment/offsite-grabpay-form.html

```diff
@@ -0,0 +1,54 @@
+<div class="payment-method"
+    data-bind="css: {'_active': (getCode() == isChecked())}, visible: isActive()">
+    <div class="payment-method-title field choice">
+        <input type="radio" name="payment[method]" class="radio"
+            data-bind="attr: {
+               'id': getCode()
+               },
+               value: getCode(),
+               checked: isChecked,
+               click: selectPaymentMethod,
+               visible: isRadioButtonVisible(),
+               enable: isActive()" />
+        <label data-bind="attr: {'for': getCode()}" class="label">
+            <span data-bind="text: getTitle()"></span>
```

Comment:
Do you mind ask PO about adding the logo 🤔 , i think we should start adding the logo for both magento and wc

Reply:
Why do you think we should start adding logos? If we do, then we should go back and add logos for all previous methods too, I think, to be consistent. But there doesn't seem to be a product requirement for it 🤔

Reply:
Is bc i seen Android and IOS will alway have the logo right ? so imo we should follow that as well

Reply:
I think there's a bit more to it than that. In this case we follow magento's standards as merchants may use our plugins alongside other payment providers. So in theory, they should be consistent with each other. But it's not a bad idea, maybe it'll be a good point to raise with product 🙂

Reply:
Sure, NP 👍